tslint.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": ["italia-tslint-rules/basic", "italia-tslint-rules/react"],
```

`react` rules are not relevant here

---

if you want you can enable the strong rules

```suggestion
  "extends": ["italia-tslint-rules/strong"],
```